### PR TITLE
Prepare Changelog for v1.2.0

### PR DESCRIPTION
**Pull request by elazzabi**
Prepare the changelog for the next version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,14 @@ If you need more information, check out our [VIP CLI documentation](https://vip.
 
 ## Changelog
 
+### 1.2.0
+
+- We now display a preview of the `sync` feature with the backup time and the search/replace taking place in your database.
+- We now display your mapped domain instead of the `vip-go.co` ones in `vip app` and `vip app list`.
+- Bug fixes.
```
Comment (mjangda) [inline README.md]: It would be nice to mention the specific fixes (if we can).

Comment (elazzabi) [reply README.md]: yes, i'll put this on hold until we finish everything for the needed for the `v1.2`

Comment (elazzabi) [reply README.md]: changed it with the help fix here #181

File: README.md [hunk 2]
```diff
@@ -20,6 +20,16 @@ If you need more information, check out our [VIP CLI documentation](https://vip.
 
 ## Changelog
 
+### 1.2.0
+
+- We now display a preview of the `sync` feature with the backup time and the search/replace taking place in your database.
+- We now display your mapped domain instead of the `vip-go.co` ones in `vip app` and `vip app list`.
+- We now display a better message when an environment do not have non-production environments.
```
Comment (nickdaugherty) [inline README.md]:
```suggestion
- We now display a better message when an app does not have any non-production environments.
```

File: README.md [hunk 3]
```diff
@@ -20,6 +20,16 @@ If you need more information, check out our [VIP CLI documentation](https://vip.
...
+- We fixed a bug where the help menu was not showing until you're logged in
```
Comment (nickdaugherty) [inline README.md]:
```suggestion
- We fixed a bug where the help menu was not showing until you're logged in.
```

File: README.md (hunk 2 above)
Comment (mjangda) [inline README.md]:
```suggestion
- We now display a better message when an environment does not have non-production environments to sync to.
```

File: README.md [hunk 4]
```diff
@@ -20,6 +20,16 @@ If you need more information, check out our [VIP CLI documentation](https://vip.
...
+- We fixed some third party vulnerabilities by updating the used packages
```
Comment (nickdaugherty) [inline README.md]:
```suggestion
- Updated dependencies.
```

File: README.md [hunk 5]
```diff
@@ -20,6 +20,16 @@ If you need more information, check out our [VIP CLI documentation](https://vip.
 
 ## Changelog
 
+### 1.2.0
+
+- We now display a preview of the `sync` feature with the backup time and the search/replace taking place in your database.
+- We now display your mapped domain instead of the `vip-go.co` ones in `vip app` and `vip app list`.
```
Comment (mjangda) [inline README.md]:
```suggestion
- We now display your mapped domain instead of the placeholder `go-vip.co` domain in `vip app` and `vip app list`.
```

**Review by nickdaugherty — DISMISSED** [review]
Looks good, left a few small grammatical suggestions and one about the dependency changes

File: README.md (hunk 3 above)
Comment (mjangda) [inline README.md]:
```suggestion
- We fixed a bug where the help menu was not showing until you're logged in.
```

File: README.md (hunk 4 above)
Comment (mjangda) [inline README.md]:
```suggestion
- We updated third party dependencies to newer, more secure versions.
```

**Review by mjangda — DISMISSED** [review]
Some minor suggestions.

**Pull request merged**